src/main/java/org/motechproject/newebodac/repository/custom/impl/BaseRepositoryImpl.java

```diff
@@ -0,0 +1,12 @@
+package org.motechproject.newebodac.repository.custom.impl;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+public abstract class BaseRepositoryImpl {
```

Comment:
you don't need the base repository, it was added in mots, because there was multiple implementations of it